plugins/ruby-foreman-google/debian/gem.list

```diff
@@ -0,0 +1,2 @@
+GEMS="https://rubygems.org/downloads/foreman_google-0.0.1.gem"
+GEMS="$GEMS https://rubygems.org/downloads/google-cloud-compute-0.2.0.gem"
```

Comment:
When I do a `gem install foreman_google`, I end up with the following gems added to my install:

```
foreman_google-0.0.1
gapic-common-0.9.0
googleapis-common-protos-1.3.12
googleapis-common-protos-types-1.3.1
googleauth-1.1.3
google-cloud-common-1.0.0
google-cloud-compute-0.3.0
google-cloud-compute-v1-1.3.0
google-cloud-core-1.6.0
google-cloud-env-1.6.0
google-cloud-errors-1.2.0
google-protobuf-3.20.1
grpc-1.46.3
```

You need to list all of them here :)